standards/bulk.md

```diff
@@ -0,0 +1,179 @@
+            }                                                            
+        }
+    ]
+}
+```
+
+- Bulk operation requests **MAY** be designated as `ATOMIC` or `ISOLATED` via the `transactionMode` field.
+    - The default transaction mode type **MUST** be `ISOLATED`.
+    - The transaction mode type **MAY** be optionally left out of the request schema in implementation.
+    - `ATOMIC` transactions will either succeed or fail together.
+    - `ISOLATED` transactions will allow for individual operations to succeed or fail independently.
+- `operationId` **MAY** be used to associate the operation in the request with the resulting operation in the response. This is useful for tracking the outcome of each operation in the response where it might be ambiguous to reference via `entityId`.
+- `action` enumeration **MUST NOT** be extended, but **MAY** be a subset of the enumeration values where not all actions are required.
+- `ifMatch` **MUST** be supported in the request schema if ETags are used for other RESTful operations on the same resource.
+- Operation collections in a request **INCLUDE** include entities of the same `id` only once. If the collection contains multiple entities with the same `id`, then the request **MUST** return a `400 Bad Request` status code and error body.
+
+#### Response
+
+- Bulk operations **MUST** include a response body schema as follows, which is not extensible:
+```json
+{
+    "status": "SUCCEEDED" | "FAILED" | "PARTIAL",                           // REQUIRED (enum): overall status of bulk operation
+    "operations": [                                                         // REQUIRED (array): results of each operation from the request.
+        {
+            "operationId": "string",                                        // REQUIRED (string): matching operation id from the request body if provided, otherwise use index value as a string
+            "action": "CREATE" | "UPDATE" | "CREATE_UPDATE" | "DELETE",     // REQUIRED (enum): repeat action type
+            "entityId": "string" | null,                                    // OPTIONAL (string): the associated id of the entity, if available
+            "entityRef": "sps-ref" | null,                                  // OPTIONAL (string): the associated sps-ref URN entity, if applicable
+            "result": {                                                     // REQUIRED (object): result of the operation
+                "status": "SUCCEEDED" | "FAILED",                           // REQUIRED (enum): status of individual operation
+                "detail": "string" | null,                                  // OPTIONAL (string): Description or detailed human-readable message about the success or failure of the operation.
+                "context": [                                                // OPTIONAL (array): List of objects providing additional context and detail on sub-reasons for any errors or failures.
```

Comment:
Is `context` specific enough, or should this be something like `errorContext`? 🤔

Reply:
the intent was to make it generic to allow for error context or otherwise if needed.

Reply:
An example from Aaron for this is using the context for something like Log Messages  with may have a valid message/code, etc that makes it useful more generically as well.